### PR TITLE
Fix issue from a rebase

### PR DESCRIPTION
**Pull request by stuartwdouglas**
It appears a rebase resulting in some code duplication

### CODE REVIEW

File: pkg/reconciler/dependencybuild/dependencybuild.go [hunk 1]
```diff
@@ -670,13 +668,21 @@ func (r *ReconcileDependencyBuild) handleBuildPipelineRunReceived(ctx context.Co
...
+				} else if i.Name == artifactbuild.PipelineResultVerificationResult {
+					verificationResults = i.Value.StringVal
 				} else if i.Name == artifactbuild.PipelineResultGavs {
 					deployed := strings.Split(i.Value.StringVal, ",")
 					db.Status.DeployedArtifacts = deployed
-				} else if i.Name == artifactbuild.PipelineResultVerificationResult {
-					verificationResults = i.Value.StringVal
+				} else if i.Name == artifactbuild.PipelineResultDeployedResources && len(i.Value.StringVal) > 0 {
+					//we need to create 'DeployedArtifact' resources for the objects that were deployed
+					deployed = strings.Split(i.Value.StringVal, ",")
```
Comment (rnc) [inline pkg/reconciler/dependencybuild/dependencybuild.go:678]: Why isn't this using `gavs` from above so that is stored in `run.Results` ?

Comment (stuartwdouglas) [reply pkg/reconciler/dependencybuild/dependencybuild.go:678]: ah, oops, I did not see your question till after I hit the merge button, this is a bug.

Comment (rnc) [reply pkg/reconciler/dependencybuild/dependencybuild.go:678]: I'll fix it in #944 :-)

Comment (rnc) [reply pkg/reconciler/dependencybuild/dependencybuild.go:678]: Ah, nm, I see you've entered https://github.com/redhat-appstudio/jvm-build-service/pull/945/

**Pull request merged**